about.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="about.css">
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/fork-awesome@1.2.0/css/fork-awesome.min.css"
+      integrity="sha256-XoaMnoYC5TH6/+ihMEnospgm0J1PM/nioxbOUdnM8HY="crossorigin="anonymous"
+      />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+      />
+    <title>About Microverse</title>
+  </head>
+  <body>
+    <div class="header" id="first-nav-bar">
+      <a href="" id="first-icons"><i class="fa fa-facebook"></i></a>
+      <a href="https://twitter.com/Faranosh_Amini" id="first-icons"><i
+          class="fa fa-twitter"></i></a>
+      <div><a href="#" id="first-icons">English</a></div>
+      <div><a href="#" id="first-icons">MyPage</a></div>
+      <div> <a href="#" id="first-icons">Logout</a></div>
+    </div>
+    <header>
+      <div><a href="index.html" class="first-logo">Microverse</a>
+      </div>
+      <nav>
+        <div class="hamburger-container">
+          <div class="close"> <img width="40px" src="Images/close.png"
+              alt="close icon" id="close"></div>
+
+          <ul class="main">
+            <li class="menu"><a href="#main-pro"
+                class="menu" id="program">Program</a></li>
+            <li class="menu"><a href=""
+                class="menu" id="speaker">Speakers</a></li>
+            <li class="menu"><a href="#partner"
+                class="menu" id="spoLOnsor">Sponsors</a></li>
+            <li class="menu"><a href="about.html"
+                class="menu" id="openAbout">About</a></li>
```

Comment:
- [ ] Kindly make sure to replace about with home as suggested in the loom video.